src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java

```diff
@@ -28,7 +28,7 @@ public class ResourceModificationEventListener {
   @TransactionalEventListener
   public void afterCreate(ResourceCreatedEvent resourceCreatedEvent) {
     log.info("ResourceCreatedEvent received [{}]", resourceCreatedEvent);
-    kafkaSender.sendSingleResourceCreated(resourceCreatedEvent.work());
+    kafkaSender.sendSingleResourceCreated(resourceRepository.getReferenceById(resourceCreatedEvent.work().getId()));
```

Comment:
We should do the same for `afterUpdate` also right?

Reply:
@pkjacob I don't think so